### PR TITLE
change to non-sensitive

### PR DESCRIPTION
**Pull request by inaod568**
Before submitting a pr:
- [x] Tests passing
- [x] Black formatting
- [x] Rebase/merge the `dev` branch
- [x] Note in the CHANGELOG

### CODE REVIEW

File: dds_cli/__main__.py [hunk 1]
```diff
@@ -648,7 +648,7 @@ def list_projects(ctx, username, json, sort, usage):
     + dds_cli.utils.multiple_help_text(item="project owner"),
 )
 @click.option(
-    "--is_sensitive",
+    "--non-sensitive",
     required=False,
     is_flag=True,
     help="Indicate if the Project includes sensitive data.",
```
Comment (alneberg) [inline dds_cli/__main__.py:654]: I would probably rephrase the help message slightly as well.

Comment (talavis) [reply dds_cli/__main__.py:654]: "Indicate whether the project contains only non-sensitive data"

**Review by alneberg — APPROVED** [review]
Great! I would think @talavis suggestion for the help message is an improvement, but no deal breaker.

File: dds_cli/__main__.py (hunk 1 above)
Comment (inaod568) [reply dds_cli/__main__.py:654]: Will fix 👍🏻

Comment (inaod568) [reply dds_cli/__main__.py]: Done

**Pull request merged**